core/arch/arm/plat-ti/conf.mk

```diff
@@ -0,0 +1,27 @@
+include core/arch/$(ARCH)/plat-$(PLATFORM)/platform_flags.mk
+
+CROSS_COMPILE	?= arm-linux-gnueabihf-
+COMPILER	?= gcc
```

Comment:
Both lines should not be needed because these are the default values set in `mk/config.mk`